src/control/server/storage/bdev/backend.go

```diff
@@ -47,6 +49,8 @@ type (
 		log     logging.Logger
 		binding *spdkWrapper
 		script  *spdkSetupScript
+
+		sync.RWMutex
```

Comment:
Nit: There's no read-locking happening, so this could just be a `sync.Mutex`. The end result is the same, but the choice helps to convey intent to the reader.